numbat/src/unit.rs

```diff
@@ -315,7 +343,7 @@ impl Unit {
 
     #[cfg(test)]
     pub fn kelvin() -> Self {
-        Self::new_base("kelvin", "K")
+        Self::new_base("kelvin", "K", AcceptsPrefix::none())
```

Comment:
It's just for tests, but I'd go with 
```suggestion
        Self::new_base("kelvin", "K", AcceptsPrefix::only_short())
```
here as well. Prefixes with `K` are definitely used, e.g. `µK`.